Userland/Libraries/LibWebView/InspectorClient.h

```diff
@@ -93,6 +95,8 @@ class InspectorClient {
     i32 m_highest_notified_message_index { -1 };
     i32 m_highest_received_message_index { -1 };
     bool m_waiting_for_messages { false };
+
+    bool m_is_windowed;
```

Comment:
All POD members should have a default initializer. It's good practice to prevent undefined behavior.